src/picologging/__init__.py

```diff
@@ -503,3 +505,42 @@ def makeLogRecord(dict):
     for k, v in dict.items():
         setattr(rv, k, v)
     return rv
+
+
+def shutdown(handlerList=None):
+    """
+    Perform any cleanup actions in the logging system (e.g. flushing
+    buffers).
+    Should be called at application exit.
+    """
+    handlers = [logger.handlers for logger in root.manager.loggerDict.values()] + [
+        root.handlers
+    ]
+    handlerList = list(itertools.chain(*handlers))
```

Comment:
There is probably a smarter way to get the list of handlers to flush/close. Any suggestions?

The standard logging uses a Weakref instance to hold refs but I guess that's not possible between C and Python types?